### PR TITLE
Fixing some basic typos in README

### PR DESCRIPTION
**Pull request by ddutta**
typos, tenses, and some cleanup

### CODE REVIEW

**Comment by ddutta** [comment]
I signed it

**Comment by ddutta** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -1,44 +1,39 @@
 # Katib
 
 Hyperparameter Tuning on Kubernetes.
-This project is [Google vizier](https://static.googleusercontent.com/media/research.google.com/ja//pubs/archive/bcb15507f4b52991a0783013df4222240e942381.pdf) inspired.
-Katib is a scalable and flexible hyperparameter tuning framework and  tightly integrate with kubernetes.
-And it does not depend on a specific DL framework.
-There are examples of three frameworks ( tensorflow, mxnet, and  pytorch).
+This project is inspired by [Google vizier](https://static.googleusercontent.com/media/research.google.com/ja//pubs/archive/bcb15507f4b52991a0783013df4222240e942381.pdf).
+Katib is a scalable and flexible hyperparameter tuning framework and is tightly integrated with kubernetes.
+Also it does not depend on a specific Deep Learning framework e.g. TensorFlow, mxnet, and  pytorch).
```
Comment (gaocegege) [inline README.md]: s/mxnet/MXNet
s/pytorch/PyTorch

Comment (ddutta) [reply README.md]: good point ... I was just cleaning basic typos ... will do it

Comment (gaocegege) [reply README.md]: Thanks :smile:

**Comment by gaocegege** [comment]
/lgtm

Thanks for your contribution!

**Pull request merged**